common/models/container.js

```diff
@@ -52,101 +56,228 @@ module.exports = function(Container) {
     // ASSUMPTION #1 we only get one file at a time
     // ASSUMPTION #2 user only uploads a valid CSV file
     var item = files[0];
+    var re = /(?:\.([^.]+))?$/;
+    var fileExtension = re.exec(item.name);
     var stream = Container.downloadStream(item.container, item.name);
 
-    var Converter = require('csvtojson').Converter;
-    var converter = new Converter({constructResult:true}); //new converter instance
+    if(fileExtension[1] == 'xls' || fileExtension[1] == 'xlsx' || fileExtension[1] == 'XLS' || fileExtension[1] == 'XLSX') {
```

Comment:
Stick to using `===` over `==`

Again, you can cover more ground if you use something like:
1. `.toLowerCase === 'value'`, or
2.  `.toUpperCase === 'VALUE') ) {`
because it lets you account for weird things like: `file.vaLue` or `file.VAlUE` etc.

I haven't tested my pseudo-code, just wrote the idea down.